onnx2pytorch/operations/loop.py

```diff
@@ -0,0 +1,187 @@
+from collections import defaultdict
+from copy import deepcopy
+from functools import partial
+import warnings
+
+import numpy as np
+import onnx
+import torch
+from onnx import numpy_helper
+from torch import nn
+from torch.nn.modules.linear import Identity
+
+from onnx2pytorch.utils import (
+    get_inputs_names,
+    get_outputs_names,
+)
+
+
+class Loop(nn.Module):
```

Comment:
Loop seems to have a lot of similarities with ConvertModel, there could be some base class that both inherit, or one inherits the other one. For now no change is necessary, but let's open an issue with regards to this after the merge.

Reply:
Sounds good, I'll be able to fix this in December / January.